build/Build.Steps.Windows.cs

```diff
@@ -177,4 +179,20 @@ partial class Build
 
             AssemblyRedirectionSourceGenerator.Generate(netFxAssembliesFolder, generatedSourceFile);
         });
+
+    Target InstallNetFxAssembliesGAC => _ => _
```

Comment:
If I execute `nuke` or `nuke Workflow` with elevated privileges it will overwrite GAC,
It will be great to have reverse step included into it, or at least provide easy way to remove all this files.

I think `nuke Workflow CleanupGAC` execution form is fine.

Reply:
I would prefer to have it as part of `nuke` or `nuke Workflow`

Reply:
There are 2 ways which might work according to your own risk.
1. Add CleanupGAC to the workflow and a warning to use it on your own risk or skip the step.
2. Add CleanupGAC as a non referenced step. You need to run it yourself manually.

Reply:
Probably point 2 is better/safer after all you said 😉 
Make it could somehow part of `nuke Cleanup`?